src/popups/ZweitausbildungPoisPopup/ZweitausbildungPoisPopup.scss

```diff
@@ -1,6 +1,8 @@
 .wkp-zweitausbildung-pois-popup {
   overflow-y: auto;
   max-height: 300px;
+  padding-right: 10px;
+  margin-right: -10px;
```

Comment:
That feels like a hack :D

Reply:
Maybe, it can be regarded as an hack. I can't access the surrounding div element in order to remove the padding for the scrollbar as I would also break all other popups.

Reply:
as I see the style is already broken if we have to do that. Let me try something

Reply:
Alternatively, we could set a max-height for the general popup with a scrollbar.

Reply:
@gevos  try with this : https://github.com/geops/trafimage-maps/pull/640

Reply:
I have cherrypicked your commit.